plutus-chain-index/README_marconi.md

```diff
@@ -138,6 +138,21 @@ Can we provide example queries?
 
 ### Installation Procedures
 
+The `marconi` executable is available as a nix flake.
+
+You may either clone the [`plutus-apps`](https://github.com/input-output-hk/plutus-apps)
+repository and run from the top-level:
+``` 
+nix build .#marconi
+``` 
+Or you may run from anywhere:
+```
+nix build github:input-output-hk/plutus-apps#marconi
```

Comment:
I am still getting the same error, even after `nix-collect-garbage -d`.

The error is:

```
┌[roa@λ] [/dev/ttys003]
└[~]> nix build github:input-output-hk/plutus-apps#marconi
error: flake 'github:input-output-hk/plutus-apps' does not provide attribute 'packages.x86_64-darwin.marconi', 'legacyPackages.x86_64-darwin.marconi' or 'marconi'
```

Reply:
I would like a way to clear the cache that is effective, before accepting this PR.

Reply:
`nix build github:input-output-hk/plutus-apps#marconi` isn't supposed to work yet because it's targeting main branch, which isn't merged with this yet. 
If you've cleared your cache try this:
`nix build github:input-output-hk/plutus-apps/zeme-iohk/flakify-the-repo#marconi`